teps/0092-scheduling-timeout.md

```diff
@@ -110,12 +124,11 @@ Other options:
   * Con: this means if the `TaskRun` is queued by the controller for processing, it would not timeout. Suggest including
```

Comment:
This is under the section "Other options", but we are proposing doing this. Maybe you could reword the other option to be starting from the creation time?